web/src/views/DandisetLandingView/Meditor.vue

```diff
@@ -240,6 +269,21 @@ export default defineComponent({
       store.commit('dandiset/setPublishDandiset', girderize(payload));
     }
 
+    const modified = ref(false);
+    const isModified = computed(() => modified.value);
```

Comment:
I think this computed variable is redundant, as it will always have the same value as `modified`. Since `modified` is a `ref`, it's also reactive. `computed` is usually used for creating reactive variables that need to modify some original data.

Reply:
Oops, fixed that.